### PR TITLE
[AWS]:Verification -ETag should not be empty during complete multipar…

### PR DESCRIPTION
**Pull request by ckulal**
…t upload

[AWS]:Verification -ETag should not be empty during complete multipart upload
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2266579

Log: http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/aws_etag_verification/aws_etag_verification.log

### CODE REVIEW

**Review by anrao19 — DISMISSED** [review]
LGTM

File: rgw/v2/tests/aws/test_aws.py [hunk 1]
```diff
@@ -70,6 +70,48 @@ def test_exec(config, ssh_con):
...
+                json.dump(mpstructure, fd)
+            complete_multipart_upload_resp = json.loads(
+                aws_reusable.complete_multipart_upload(
+                    bucket_name, key_name, "test.json", upload_id, endpoint
+                )
+            )
+            if not complete_multipart_upload_resp["ETag"]:
+                raise AssertionError(
+                    "Etag not generated during complete multipart upload operation"
+                )
```
Comment (hmaheswa) [inline rgw/v2/tests/aws/test_aws.py:114]: @ckulal , could you place this code in a separate method "upload_multipart_object" for reusability and take the object size from config 
as present in https://github.com/red-hat-storage/ceph-qe-scripts/blob/master/rgw/v2/tests/s3_swift/reusable.py#L450

File: rgw/v2/tests/aws/reusable.py [hunk 2]
```diff
@@ -75,6 +75,132 @@ def list_object_versions(bucket_name, end_point, ssl=None):
...
+    Ex: /usr/local/bin/aws s3api create-multipart-upload --bucket <bucket_name> --key <key_name> --endpoint <endpoint_url>
+    Args:
+        bucket_name(str): Name of the bucket
+        key_name(str): Name of the object for which multipart upload has to be initiated
+        end_point(str): endpoint
+        ssl:
+    Return:
+        Response of create-multipart-upload
+    """
+    list_method = AWS(operation="create-multipart-upload")
```
Comment (hmaheswa) [inline rgw/v2/tests/aws/reusable.py:90]: @ckulal could you use the proper variable name like aws_conn in all methods 
list_method is misleading

Comment (ckulal) [reply rgw/v2/tests/aws/reusable.py]: done

File: rgw/v2/tests/aws/test_aws.py (hunk 1 above)
Comment (ckulal) [reply rgw/v2/tests/aws/test_aws.py:88]: Done.
New log: http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/aws_etag_verification/aws_etag_verification.log-new

**Review by hmaheswa — APPROVED** [review]
LGTM

**Review by TejasC88 — APPROVED** [review]

**Pull request merged**